src/content/admin/dashboard.mdx

```diff
@@ -207,4 +209,17 @@ If you choose to pull images from a private registry in addition to [Okteto's Co
 
 Dynamically configure Okteto's connection to AWS and GCP, eliminating the need for developers to manage static credentials.
 
-[Review this page for a full description of how to configure and start using Cloud Credentials →](admin/cloud-credentials/index.mdx)
\ No newline at end of file
+[Review this page for a full description of how to configure and start using Cloud Credentials →](admin/cloud-credentials/index.mdx)
+
+## Garbage Collector
+
+Configure sleep and delete periods for Okteto Developer Environments and Preview Environments from the UI.
```

Comment:
Same as before, today, "Developer Environments" is not used across product/docs

Reply:
Thanks, will update both of these